docs/reference/docs/bulk.asciidoc

```diff
@@ -66,17 +66,97 @@ example of a correct sequence of bulk commands:
 
 [source,js]
 --------------------------------------------------
+POST _bulk
 { "index" : { "_index" : "test", "_type" : "type1", "_id" : "1" } }
 { "field1" : "value1" }
 { "delete" : { "_index" : "test", "_type" : "type1", "_id" : "2" } }
 { "create" : { "_index" : "test", "_type" : "type1", "_id" : "3" } }
 { "field1" : "value3" }
-{ "update" : {"_id" : "1", "_type" : "type1", "_index" : "index1"} }
+{ "update" : {"_id" : "4", "_type" : "type1", "_index" : "index1"} }
 { "doc" : {"field2" : "value2"} }
 --------------------------------------------------
+// CONSOLE
+
+The result of this bulk operation is:
 
-In the above example `doc` for the `update` action is a partial
-document, that will be merged with the already stored document.
+[source,js]
+--------------------------------------------------
+{
+   "took": ...,
```

Comment:
I typically leave `took` with whatever value it returned on my laptop and use `s///` to replace it with the response. This means the docs contain a realistic looking number.